tests/rest/test_rest.py

```diff
@@ -467,11 +492,12 @@ def main():
     node_ext_address_test(200)
     node_state_test(200)
     node_network_name_test(200)
+    node_state_test_attached()  # wait for attached state
     node_leader_data_test(200)
     node_num_of_router_test(200)
     node_ext_panid_test(200)
     node_coprocessor_version_test(200)
-    diagnostics_test(20)
+    # diagnostics_test(20)  # partly replaced with restjsonapi tests
```

Comment:
Is `/diagnostics` removed? if not, should we keep the test?

If the API behavior changed, will it break anything?

Reply:
/diagnostics is removed and javascript for the topology browser is updated. 
"partly" refers to potential for increasing test coverage in restjsonapi tests.